### PR TITLE
Fix data parsing, update data

### PR DESCRIPTION
**Pull request by frebib**
Data consistency apparently isn't NHS' strong-suit. Below are some of
the anomalies that broke the parser.

    1st dose5
    Total 1st Doses6
    2nd dose5
    Total 2nd Doses6
    Cumulative Total Doses to Date6
    % who have had at least 1 dose (using ONS denominators)7
    % who have had both doses (using ONS denominators)7

Fixed by relaxing the absolute string comparison instead for a "close
enough" string-contains approach.

Signed-off-by: Joe Groocock <me@frebib.net>

### CODE REVIEW

File: data/parse.py [hunk 1]
```diff
@@ -133,24 +133,28 @@ def is_nan(cell) -> bool:
...
                 continue
-            if dose in ["Total 1st Doses", "Total 2nd Doses"]:
-                # Ignore dose summaries.
-                continue
 
-            is_dose_and_group_all = dose == "Cumulative Total Doses to Date"
+            is_dose_and_group_all = "cumulative total doses to date" in dose.lower()
 
-            if dose == "1st dose":
+            if "1st dose" in dose.lower():
```
Comment (mishajw) [inline data/parse.py]: Remove the substring matching? Unknown column titles that aren't in `ignore` but contain the string `"1st dose"` will lead to unhandled errors in the data. Better crash than have something incorrect

**Review by mishajw — CHANGES_REQUESTED** [review]
❤️

File: data/parse.py (hunk 1 above)
Comment (frebib) [reply data/parse.py]: For some reason they renamed the fields in this sheet to `1st dose5`. Should I just allow that as a static allowed value instead?

Comment (frebib) [reply data/parse.py]: This works, I'll update
```diff
diff --git a/data/parse.py b/data/parse.py
index e8e0d19..a822700 100644
--- a/data/parse.py
+++ b/data/parse.py
@@ -152,9 +152,9 @@ def __parse_df_weekly(source: Source, df: pd.DataFrame) -> Iterable[Vaccinated]:

             is_dose_and_group_all = "cumulative total doses to date" in dose.lower()

-            if "1st dose" in dose.lower():
+            if dose in ["1st dose", "1st dose5"]:
                 dose = Dose.DOSE_1
-            elif "2nd dose" in dose.lower():
+            elif dose in ["2nd dose", "2nd dose5"]:
                 dose = Dose.DOSE_2
             elif is_dose_and_group_all:
                 dose = Dose.ALL
```

**Review by mishajw — APPROVED** [review]

**Pull request merged**